modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -441,7 +441,7 @@ public String toOperationId(String operationId) {
             operationId = "call_" + operationId;
         }
 
-        return camelize(operationId);
+        return camelize(operationId).replace(" ", "");
```

Comment:
I'm surprised that camelize doesn't do this for us.

Looking at the code ( https://github.com/OpenAPITools/openapi-generator/blob/0fc2df6a4714343a9df621c8a5e6b767fe81a2cf/modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/StringUtils.java ), I think we might be better doing:

`return camelize(underscore(operationId));`

which I think will handle both issues.

Reply:
Using camelize(underscore(operationId)) is not oK.
The example case will get the method "GetRepoinfo", Not the wanted "GetRepoInfo".

 **CreateRepo**](repo_api.md#CreateRepo) | **POST** /repos | 
-**GetRepoInfo**](repo_api.md#GetRepoInfo) | **GET** /repos/{repoId} | 
+**GetRepoinfo**](repo_api.md#GetRepoinfo) | **GET** /repos/{repoId} |